Makefile

```diff
@@ -21,7 +24,12 @@ build:
 	git submodule init
 	git submodule update --init --recursive
 	# Update RapidWright jars
-	pushd third_party/RapidWright && make update_jars && popd
+	pushd ${RAPIDWRIGHT_PATH} && \
+		make update_jars && \
+		wget https://github.com/Xilinx/RapidWright/releases/download/v2020.2.5-beta/rapidwright_api_lib-2020.2.5-patch1.zip && \
```

Comment:
Is this necessary? Aren't all the changes available on RapidWright GH ?

Reply:
Yes, I'll add a TODO comment to point to the RW issue